schema/release-schema.json

```
@@ -27,7 +27,7 @@
     },
     "tag": {
       "title": "Release Tag",
-      "description": "One or more values from the open [releaseTag](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#release-tag) codelist. Tags can be used to filter releases and to understand the kind of information that releases might contain.",
+      "description": "One or more values from the open [releaseTag](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#release-tag) codelist. Tags can be used to filter releases and to understand the kind of information that releases might contain. It is encouraged to use tags from the releaseTags codelist whenever possible, but additional ones can be used based on specific use cases such as mapping with a particular notice, form or event.",
```

I think we're trying to reduce publisher-specific instructions in the schema, so that it's useful to both publishers and users.

We can instead put publisher instructions at https://standard.open-contracting.org/latest/en/schema/codelists/#release-tag Note that the general policy about using existing codes is already expressed at the top of that page.

For the schema, to assist users in interpreting additional codes, we can do:

```suggestion
      "description": "One or more values from the open [releaseTag](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#release-tag) codelist. Tags can be used to filter releases and to understand the kind of information that releases might contain. Codes outside the releaseTag codelist might indicate, for example, the notice or form to which the release corresponds, or the event that triggered the publication of the release.",
```